src/components/connection_handler/test/connection_handler_impl_test.cc

```diff
@@ -1194,13 +1213,13 @@ TEST_F(ConnectionHandlerTest, SessionStop_CheckSpecificHash) {
 
     const uint32_t end_audio_wrong_hash =
         connection_handler_->OnSessionEndedCallback(
-            uid_, start_session_id_, &wrong_hash, kRpc);
+            uid_, out_context_.new_session_id_, wrong_hash, kRpc);
```

Comment:
Use `&wrong_hash`

Reply:
@jacobkeeler fixed in https://github.com/smartdevicelink/sdl_core/pull/1971/commits/c95042bdce0bc3d598ce8300e7b17d7d55e9e4b4